fork.yaml

```diff
@@ -0,0 +1,30 @@
+title: "celo - go-ethereum fork diff overview"
```

Comment:
Do we want this file in the root directory? We can put it somewhere also to avoid clutter in the root dir.